projects/igniteui-angular/src/lib/core/styles/components/grid-toolbar/_grid-toolbar-component.scss

```diff
@@ -23,6 +23,10 @@
         @extend %igx-grid-toolbar__actions !optional;
     }
 
+    @include e(export-progress){
```

Comment:
Can we name this something more generic. I think it might be used for other purposes as well.

Reply:
How about **e(progress-bar)**?

Reply:
Yeah, I think that's fine :+1: